jaxrs-clients/build.gradle

```diff
@@ -50,8 +55,13 @@ shadowJar {
     relocate('zipkin', 'httpremoting.shaded.zipkin')
     relocate('org.joda.time', 'httpremoting.shaded.org.joda.time')
     relocate('okio', 'httpremoting.shaded.okio')
+    relocate('org.jvnet.animal_sniffer', 'httpremoting.shaded.org.jvnet.animal_sniffer')
```

Comment:
think this is probably related to the agentloader of the alpn thing, in whcih case also want to exclude it